sefaria/helper/schema.py

```diff
@@ -269,30 +270,45 @@ def change_node_title(snode, old_title, lang, new_title):
     pass
 
 
-def replaceBadNodeTitles(title, bad_char, good_char, lang):
+def change_char_node_titles(index_title, bad_char, good_char, lang):
+    '''
```

Comment:
We need tests for this function.  See the tests for other functions in this module.